lib/event.rb

```diff
@@ -0,0 +1,51 @@
+class Event
+  attr_reader :name, :food_trucks
+
+  def initialize(name)
+    @name = name
+    @food_trucks = []
+  end
+
+  def add_food_truck(food_truck)
+    @food_trucks << food_truck
+  end
+
+  def food_truck_names
+    @food_trucks.map do |food_truck| 
+      food_truck.name
+    end
+  end
+
+  def food_trucks_that_sell(item)
+    @food_trucks.find_all do |food_truck|
+      food_truck.inventory.include?(item)
+    end
+  end
+
+  def sorted_item_list 
+    @food_trucks.flat_map do |food_truck|
+      food_truck.inventory.map do |item, amount| 
+       item.name
+      end
+    end.uniq.sort
```

Comment:
Yup, nice job! And I know you would have caught this right away if your test had been accurate!